OpenArtemis/Scrape/HeadlessWebManager.swift

```diff
@@ -24,20 +24,48 @@ class HeadlessWebManager: NSObject, WKNavigationDelegate {
     
     // Adjusted to store the value of autoClickExpando
     func loadURLAndGetHTML(url: URL, autoClickExpando: Bool = false, preventCacheClear: Bool = false, completion: @escaping (Result<String, Error>) -> Void) {
-        self.completion = completion
-        self.shouldAutoClickExpando = autoClickExpando  // Store the flag value
+        let urlSchemeReplacementResult = addSchemeToUrl(url)
         
-        let request = URLRequest(url: url)
-        if !preventCacheClear {
-            clearWebCache { [weak self] in
-                guard let self = self else { return }
+        switch urlSchemeReplacementResult {
```

Comment:
Note: shouldn't be any changes here aside from getting a url using addSchemeToUrl, all of the removals/additions should just be to accomodate new indentations.